core/arch/arm/mm/tee_pager.c

```diff
@@ -963,11 +963,14 @@ static bool tee_pager_unhide_page(vaddr_t page_va)
 					page_va, a);
```

Comment:
By the way, can you remove this trace. It's old debugging i guess. Looks ugly. (I saw it after the tlb refine was merged but forgot it since)